packages/cli/src/hooks/commandNotFound/commandNotFound.ts

```diff
@@ -0,0 +1,10 @@
+import {Hook} from '@oclif/config';
+import {buildEvent} from '../analytics/eventUtils';
+
+const hook: Hook<'command_not_found'> = async function (options) {
+  this.config.runHook('analytics', {
+    event: buildEvent('invalid command', {command: options.id}),
```

Comment:
@tcamire, here is an event name that does not really fit the proposed syntax . Do you have any recommendation to flag invalid commands?

Reply:
That works with me.